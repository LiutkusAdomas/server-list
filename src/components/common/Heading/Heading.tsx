import { ReactNode } from 'react';

type HeadingPropsType = {
  children?: ReactNode;
};

export const Heading: React.FC<HeadingPropsType> = ({ children }) => {
  return <h2 className="text-center text-2xl font-extrabold text-gray-900">{children}</h2>;
};
