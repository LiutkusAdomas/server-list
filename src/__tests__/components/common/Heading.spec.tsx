import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { Heading } from '../../../components/common/Heading/Heading';

describe('Heading component', () => {
  it('should render a heading element with text', () => {
    const { asFragment } = render(<Heading>Test</Heading>);
    expect(asFragment).toMatchSnapshot();
  });
});
