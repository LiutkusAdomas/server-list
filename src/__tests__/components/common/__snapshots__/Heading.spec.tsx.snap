// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Heading component > should render a heading element with text 1`] = `[Function]`;
